Markdown Cleanup 2020-12-02

Simran-B opened the pull request:
- Wrap long lines
- Blank lines above and below fenced code blocks (except in lists, because that leads to increased padding for the entire list)
- Fix indentation of code blocks in lists
- Add link to Slack
- Improve existing link
- Fix spelling of some words

jfpanisset commented on CHANGELOG.md:
```diff
@@ -220,9 +220,9 @@ All notable changes to this project will be documented in this file.
 * `aswfdocker packages` lists all ci packages.
 * `aswfdocker images` lists all ci images.
-* `aswfdocker migrate` allows migration of docker packages from one organisation to another.
+* `aswfdocker migrate` allows migration of docker packages from one organization to another.
```
Arguably both organization and organisation is OK in UK English, but only organization is OK in US English, so the zeds have it (right?).

Simran-B replied:
My rationale is that the ASWF as a US legal body should use US spelling, but it's only a suggestion after all.

jfpanisset commented on README.md:
```diff
@@ -50,57 +50,92 @@ Each image (apart from `ci-common`) is available for multiple VFX Platform Years
+[VFX Platform](https://vfxplatform.com) compliant images. And a preview of 2021
+with devtoolset-9.1 (GCC-9.3.1) and clang-10, N.B. that the built-in CUDA-10.2
+is currently incompatible with GCC-9, hoping to update to CUDA-11 as soon as
+possible to restore CUDA support in 2021 images.
```
Isn't this done now with latest aswf-docker version?

Simran-B replied:
From the changelog:
> VFX 2021 images are now built against CUDA-11.1.
So I guess this remark can be removed.

jfpanisset commented on README.md:
```diff
@@ -111,15 +146,18 @@ aswfdocker build -n aswftesting/ci-package-usd:2019
 ### Images
-Images can be build with recent Docker versions but do not require [buildx](https://docs.docker.com/buildx/working-with-buildx/)
-but it is recommended to speed up large builds.
+Images can be build with recent Docker versions but do not require
```
Should be "Images can be built"

jfpanisset commented on python/aswfdocker/dockergen.py:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 """
-Migration of ASWF docker images between docker organisations
+Migration of ASWF docker images between docker organizations
```
Should Docker be capitalized?

jfpanisset commented on python/aswfdocker/migrater.py:
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 """
-Migration of ASWF docker images between docker organisations
+Migration of ASWF docker images between docker organizations
```
Should Docker be capitalized?

jfpanisset reviewed (dismissed):
All looks good to me, a few minor comments for stuff I noticed that isn't changed by this PR.

Simran-B commented:
I addressed the review comments and Improved a few more things.

There are two minor inconsistencies left:
- OSL is referred to as OpenShadingLanguage as well as Open Shading Language. The latter is the correct name, but the former is the repository name. In some places the abbreviation OSL is used instead of the full name. Not sure which one or if both should be used in the container metadata for instance.
- USD is called USD throughout, but in the image description it's `USD (Universal Scene Description)`. Should the abbreviation be used in the title, and `abbreviation (full name)` in the description for all images?

aloysbaillet reviewed (approved):
Thanks @Simran-B ! I agree with using US English more consistently, I'm guilty of mixing it with Australian...

jfpanisset reviewed (approved):
Nice cleanups, thank you. Couple of comments:

* will fixing capitalization for the title and description of containers end up invalidating currently published containers?
* outside the scope of this PR, but there seems to be a lot of duplication in the README.md files for the various containers, would it make sense to somehow template those, or at least have a script to regenerate them automatically, since I imagine it still makes sense to have those checked in to the repo for documentation purposes?

Simran-B commented:
> will fixing capitalization for the title and description of containers end up invalidating currently published containers?

Hm, it's doesn't change functionality so it's logically not invalidating them, but every change to a Dockerfile results in an image with a different ID so it's technically a different image because of the capitalization changes... Shall I split this PR into two, with all changes that affect the image ID in a new one that can be merged later when images need to be updated anyway?
